app/services/validator_check_active_service.rb

```diff
@@ -99,6 +78,7 @@ def acceptable_stake?(validator)
     with_acceptable_stake.exists?
   end
 
+  # check if validator had no delinquent history since DELINQUENT_TIME
   def non_delinquent_history_exists?(validator)
     ValidatorHistory.where(account: validator.account, delinquent: false)
```

Comment:
Can we use `validator_histories_for_validator` method here as well? Or edit validator_histories_for_validator method so that it can be reused here?